psalm-baseline.xml

```diff
@@ -143,7 +143,8 @@
     </PropertyNotSetInConstructor>
   </file>
   <file src="src/Adapter/Http.php">
-    <DocblockTypeContradiction occurrences="4">
+    <DocblockTypeContradiction occurrences="5">
+      <code>$auth === false</code>
```

Comment:
@Ocramius this will lead to the same behavior, see https://3v4l.org/Ne6IY

Reply:
Ah, because `base64_decode(false)` acting as a protection here: https://3v4l.org/EN7lP